<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D_add_lk</class>
 <widget class="QDialog" name="D_add_lk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="10,7,0">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_tlg">
       <property name="text">
        <string>Номер указания/ТЛГ</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="tlgEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="date_tlg">
       <property name="text">
        <string>Дата</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="tlgDateEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Label">
       <property name="text">
        <string>Срок выполнения</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="tlgDeadlineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Label_2">
       <property name="text">
        <string>Краткое содержание</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Label_3">
       <property name="text">
        <string>Номер листа контроля</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lcEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="spec_groupbox">
       <property name="title">
        <string>Специальности</string>
       </property>
       <layout class="QHBoxLayout" name="spec_layout"/>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="podr_groupbox">
       <property name="title">
        <string>Подразделения</string>
       </property>
       <layout class="QHBoxLayout" name="unit_layout"/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
